LadderManager.cpp

```diff
@@ -226,43 +228,64 @@ bool LadderManager::SaveReplay(sc2::Connection *client, const std::string& path)
 }
 
 
-void StartBotProcess(std::string CommandLine)
+void StartBotProcess(BotConfig Agent, std::string CommandLine)
 {
```

Comment:
I'm not sure its worth passing the entire BotConfig structure in to just log out a message.  At the very least we should pass it as const to avoid copying.  It might be better to either log from the calling function, or just log the command line.

Reply:
We also need it for the working directory.

Const OK. Probably const reference.

IMO is it worth it because otherwise the log message can be fragmented due to the async call. And in the end it is not really a bottleneck.

Reply:
yeh, ok.  I'll merge it in and make the changes

Reply:
Const doesn’t help without a reference.